bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -518,60 +518,69 @@ export class MapModule extends AbstractMapModule {
     /**
      * Formats the measurement to ui.
      * Returns a string with the measurement and
-     * an appropriate unit (m/km or m²/ha/km²)
+     * an appropriate unit (m/km or m²/ha/km² or M (m/km))
      * or an empty string for point.
-     *
+     * 
      * @public @method formatMeasurementResult
-     *
+     * 
      * @param  {number} measurement
      * @param  {String} drawMode
      * @param  {Number} fixedDigits (optional)
+     * @param  {String} format (optional) - e.g. 'nauticalMiles'
      * @return {String}
-     *
      */
-    formatMeasurementResult (measurement, drawMode, fixedDigits) {
+    formatMeasurementResult (measurement, drawMode, fixedDigits, format) {
         if (typeof measurement !== 'number') {
             return;
         }
-        let result;
-        let unit;
-        let decimals;
+        const METERS_PER_NAUTICAL_MILE = 1852;
         const zoomedForAccuracy = this.getResolution() < 1;
 
+        // helper to format number with decimal separator and chosen digits
+        const fmt = (value, digits) => value.toFixed(digits).replace('.', Oskari.getDecimalSeparator());
+
+        // default format for line (m / km)
+        const defaultLineFormat = (measurement, fixedDigits) => {
```

Comment:
We don't really need to define the `defaultLineFormat()`  here, since the function can just call itself again with different parameters to get the "defaultLineFormat"? With this implementation we are constantly redefining the function without the functionality changing. It's unnecessary and leads to more memory consumption that is needed.